### ISSUE TITLE
Please swap step 23 with step 22 OR make a WARNING to check step 23 before booting

### ISSUE THREAD

**Issue by wdiane1** [NetAppDocs/ontap-systems-upgrade#32]
Page: [Set the FC or UTA/UTA2 configuration on node3](https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade-arl-auto-app/set_fc_or_uta_uta2_config_on_node3.html)

**Comment by netapp-pcarriga** [comment]
@wdiane1 Thank you for your feedback!
I'll have your recommendation verified and then update the documentation.

**Comment by netapp-pcarriga** [comment]
@wdiane1 - We've added a note in the [Check and configure UTA/UTA2 ports on node3](https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade-arl-auto-app/set_fc_or_uta_uta2_config_on_node3.html#check-and-configure-utauta2-ports-on-node3) section after step 21 with instruction on how to proceed depending on your system configuration. Where required, we have made the equivalent update in the "Check and configure UTA/UTA2 ports" sections in the AFF and FAS system upgrade documentation.
Closing this issue and thanks again for you feedback!
